plugins/filter_map.rb

```diff
@@ -0,0 +1,90 @@
+#
```

Comment:
https://github.com/AnchorFree/php-logger/pull/8/files#diff-3254677a7917c6c01f55212f86c57fbfR15
is enough